### DISCUSSION TITLE
Problem with the verification of institunional Email domain

### DISCUSSION THREAD

**Discussion by GhaemiBafghi** [RosettaCommons/rosetta#254] [ROSIE]
Hi. I have verified the institutional Email domain but still ROSIE message: "Your GitHub account does not have a verified academic email address!''. Can you help me with it?
My academic email domain is @um.ac.ir.

**Comment by roccomoretti** [comment]
Unfortunately, there are legal issues with using servers in the United States to provide computational services to people in Iran, which means that we are unable to approve Iranian email addresses for ROSIE accounts at this time.

We apologize for the current state of the world.

**Comment by GhaemiBafghi** [comment]
Dear Rocco Moretti

Thanks for your reply.

On 2024-11-27 19:02, Rocco Moretti wrote:

> Unfortunately, there are legal issues with using servers in the United 
> States to provide computational services to people in Iran, which means 
> that we are unable to approve Iranian email addresses for ROSIE 
> accounts at this time.
> 
> We apologize for the current state of the world.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>



Links:
------
[1] 
https://github.com/RosettaCommons/rosetta/discussions/254#discussioncomment-11396275
[2]
